### PR TITLE
[SYCL] Mark exclusive_scan_over_group.cpp as UNSUPPORTED: ze_debug4

### PR DESCRIPTION
**Pull request by aelovikov-intel**

### CODE REVIEW

File: SYCL/GroupAlgorithm/exclusive_scan_over_group.cpp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // UNSUPPORTED: cuda || hip
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
+// UNSUPPORTED: ze_debug4
```
Comment (bso-intel) [inline SYCL/GroupAlgorithm/exclusive_scan_over_group.cpp]:
```suggestion
// UNSUPPORTED: ze_debug4,ze_debug-1
```

Comment (bso-intel) [reply SYCL/GroupAlgorithm/exclusive_scan_over_group.cpp]: ZE_DEBUG=-1 includes ZE_DEBUG=4.
So, please mark both ze_debug-1 and ze_debug4.

**Review by bso-intel — CHANGES_REQUESTED** [review]

File: SYCL/GroupAlgorithm/exclusive_scan_over_group.cpp (hunk 1 above)
Comment (aelovikov-intel) [reply SYCL/GroupAlgorithm/exclusive_scan_over_group.cpp]: Argh, I misread it originally - thought that it was a positive "one" with different (for some reason) spelling. Now it makes sense.

Thanks!

**Review by bso-intel — APPROVED** [review]

**Comment by aelovikov-intel** [comment]
@bso-intel , the failures must be unrelated to the change and we don't want to stress the infrastructure with rebase/retest, I think.

Can you please merge this change in (I don't have write access)?

**Comment by bso-intel** [comment]
I don't have write permission either.
@intel/llvm-reviewers-runtime 
Please merge this PR.
The failure seems unrelated to this changeset.

**Comment by steffenlarsen** [comment]
Failures have been fixed with https://github.com/intel/llvm-test-suite/pull/976 and are indeed unrelated to this.

**Review by steffenlarsen — APPROVED** [review]

**Pull request merged**